<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Buy_Books</class>
 <widget class="QWidget" name="Buy_Books">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>309</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
                          background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(212, 146, 70, 255), stop:1 rgba(255, 255, 255, 255));
                          border-style: inset;
                          border-style: outset;
                          border-radius: 10px;
                          border-color: beige;
                          font: bold 14p;
                          min-width: 10em;
                          padding: 6px;
}
QPushButton:pressed{
                          background-color: rgb(224, 200, 0);
                          border-style: inset;
                          color: red;
                          border-style: outset;
                          border-width: 3px;
                          border-radius: 10px;
                          border-color: beige;
                          font: bold 14p;
                          min-width: 10em;
                          padding: 6px;
}
QDialog
{
 background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(50, 88, 99, 255), stop:1 rgba(255, 255, 255, 255))
}
 QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: rgb(85, 170, 127);
 }
QGroupBox::title {
  subcontrol-position: top center; /* position at the top center */
}
QGroupBox{
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QLabel
{
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QTableWidget {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0.272727 rgba(203, 228, 225, 255))
}
QHeaderView, QHeaderView::section {
    background-color: rgba(128, 128, 128, 128);
}</string>
  </property>
  <widget class="QTableWidget" name="Books_ordertable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>130</number>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Delete_button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Buy_button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BUY</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>211</width>
     <height>47</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLineEdit" name="PhoneNumber_lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Number telephone</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>211</width>
     <height>47</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>–êddress</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="Address_line"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="nameuser_label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_order">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
